timewatcher/timewatcher.py

```diff
@@ -14,6 +14,9 @@
 from . import consts
 from . import data
 
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
Why do we need this? this package doesn't really use a `logger` so we don't see the urlib3 logs anyway.. (?)

Reply:
We can remove it, it just for me logging